web/client/themes/default/ms-variables.less

```diff
@@ -37,6 +37,7 @@
 
 @ms-main-color: @ms2-color-text;
 @ms-main-bg: @ms2-color-background;
+@ms-mask-bg: #00000088;
```

Comment:
This variable is not used please remove it

Reply:
it's used